engine/collection/epochmgr/engine.go

```diff
@@ -179,7 +182,13 @@ func (e *Engine) checkShouldStartPreviousEpochComponentsOnStartup(engineCtx irre
 	}
 	finalizedHeight := finalHeader.Height
 
-	prevEpoch := finalSnapshot.Epochs().Previous()
+	prevEpoch, err := finalSnapshot.Epochs().Previous()
+	if err != nil {
+		if errors.Is(err, protocol.ErrNoPreviousEpoch) {
```

Comment:
we have a similar check 7 lines below:
https://github.com/onflow/flow-go/blob/625c9244b03052fa7af5d05a4fa3a7a515c9bdba/engine/collection/epochmgr/engine.go#L194-L196
If I am understanding correctly, one of those checks should be removed in the next PR?